### DISCUSSION TITLE
How can I check history of pull-requests

### DISCUSSION THREAD

**Discussion by uneoka0114** [Finalgoal231/.github#18] [Q&A] [answered]
Hello, everyone!
I work on my repository.
I have sent pull-request from "dev" branch to "main".
And I have merged that successfully.
But it doesnt show my work history such as graph of redume.
So, I wanna check history of pull-request.
Please send me good solutions.
Thank you, very much.

**Comment by popstar7** [comment, marked as answer]
When you reach out at Prull requests tab, you can see your PRs there.
But MERGED PRs was CLOSED, so you need to clear search input data to see your previous PRs.
